File: sem 3/python labbbb/Day 3/40_Program2.py
n = int(input("Enter a number "))
c = 0
d = 1
if(n <= 1):
    print("Not Prime")
else:
    for i in range(2, n, 1):
        if (n % i) == 0:
            c = c+1
    if(c == 0):
        print("Prime")
    else:
        print("Not Prime")

File: sem 2/computer lab 7th july/40_program2.c
#include<stdio.h>
void callByValue(int a)
{
a=a+6;//no change occurs 
}
void callByReference(int *b)
{
*b=*b+6;//change occurs in a of main()
}
int main()
{
    printf("Enter 1 number\n");
    int a;
    scanf("%d",&a);
    printf("Before call by value, a is %d\n",a);
    callByValue(a);
    printf("After call by value, a is %d\n",a);
    printf("Before call by reference, a is %d\n",a);
    callByReference(&a);
    printf("After call by reference, a is %d\n",a);
}